packages/admin/dashboard/src/hooks/api/products.tsx

```diff
@@ -87,9 +87,14 @@ export const useDeleteProductOption = (
 export const useProductVariant = (
```

Comment:
We need to add the same fix here you did for for product (include the `query` in query key).
If you open edit variant from product and then go to variant page it will break since pulling the variant from the same cache key.